docs/ref/models/querysets.txt

```diff
@@ -2230,13 +2230,18 @@ instances, generally with one query::
     >>> objs[0].headline = 'This is entry 1'
     >>> objs[1].headline = 'This is entry 2'
     >>> Entry.objects.bulk_update(objs, ['headline'])
+    2
+
+.. versionchanged:: 4.0
+
+    Return value with the number of updated objects was added.
```

Comment:
It reads fine to me as either `objects updated` -- as Mariusz suggested -- or `updated objects`. I don't mind which as long as it is applied consistently which is not currently the case.

```suggestion
    The return value of the number of objects updated was added.
```

Reply:
I think the phrase `updated objects` is used now throughout this patch now. please let me now, if that's not the case.

Reply:
No, it's not. In the release note we have `updated objects` in the new caveat `objects updated` etc.

Please use `updated objects` everywhere.

Reply:
sorry, i just copy pasted Nick's suggestion. I have updated it.